apps/coscrad-frontend/src/components/loading/loading.tsx

```diff
@@ -1,21 +1,42 @@
-import { useEffect, useState } from 'react';
-
-const MAX_NUMBER_OF_DOTS = 3;
-
-const DOT_RATE = 1500; // rate dots are added in milliseconds
-
-const DOT_CHAR = '.';
-
-type LoadingState = number;
+import { Box, CircularProgress, Typography } from '@mui/material';
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
 
 export const Loading = (): JSX.Element => {
-    const [numberOfDots, setNumberOfDots] = useState<LoadingState>(0);
+    const { loadingMessage } = useContext(ConfigurableContentContext);
+
+    const { organizationLogoUrl } = useContext(ConfigurableContentContext);
 
-    useEffect(() => {
-        setTimeout(() => {
-            setNumberOfDots((numberOfDots + 1) % (MAX_NUMBER_OF_DOTS + 1));
-        }, DOT_RATE);
-    }, [numberOfDots]);
+    return (
+        <Box
+            sx={{
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'center',
+                justifyContent: 'center',
+                width: '100%',
+                padding: '3em 0 0 0',
+            }}
+            data-testid="loading"
+        >
+            <Box style={{ position: 'relative', display: 'inline-block' }}>
```

Comment:
Just noticing the use of style.  Maybe use a `styled` component here for consistency.  Or can you accomplish this using `sx`?